src/connection.rs

```diff
@@ -613,6 +613,24 @@ impl Connection {
     /// # Ok::<(), Error>(())
     /// ```
     ///
+    /// By default, a maximum of 2 rows are returned when the query is first
+    /// executed. To modify this, use `StmtParam::PrefetchRows(u32)` to customize
+    /// it. For more information on the difference between this and `FetchArraySize`,
+    /// see [this writeup](https://blog.dbi-services.com/arraysize-or-rowprefetch-in-sqlplus/).
```

Comment:
Another reference is https://oracle.github.io/node-oracledb/doc/api.html#rowfetching